src/hidra/receiver/plugins/asapo_producer.py

```diff
@@ -292,6 +292,11 @@ def _parse_file_name(self, path):
 
         stream = get_entry(matched, "scan_id")
         file_idx = int(get_entry(matched, "file_idx_in_scan"))
+        try:
+            idx_offset = get_entry(matched, "file_idx_offset")
```

Comment:
Doesn't this need to be converted to `int` first?

Reply:
Done. Corresponding warning message in case of problem is introduced.